### ISSUE TITLE
[FEATURE] - Set opacity for separators

### ISSUE THREAD

**Issue by shayded-exe** [alefragnani/vscode-separators#77]
It would be great to be able to set the opacity of the separators without overriding the color.

**Comment by alefragnani** [comment]
Hi @shayded-exe ,

I suppose you mean a single setting, to be applied to all available colors, right?

AFAIK, unfortunately, it seems this is not doable in VS Code, yet. At least not this approach to _combine_ colors available in `workbench.colorCustomizations`.

I'll do a bit of research on it, to confirm if this is doable or not.

Hope this helps

**Comment by shayded-exe** [comment]
Thanks.

I see that VS Code doesn't offer an API yet for getting a theme color as hex, which is surprising. https://github.com/microsoft/vscode/issues/32813

**Comment by alefragnani** [comment]
Exactly, this issue is the one I'm tracking for a while, but unfortunately, no ETA.

I looked for some extensions and settings that could replicate your request, with no success. Looking for reasonable alternatives also without success. So, I guess this won't happen, for now.

I'll close this issue, since I don't see this happening in the near future. If something changes on that issue, I'll be notified, because I've subscribed to it.

Hope this helps
